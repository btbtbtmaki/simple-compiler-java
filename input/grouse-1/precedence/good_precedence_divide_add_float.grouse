main {

  print 7.6 / 3.8 + 3.6 nl;
  print 5.3 / 6.9 + 6.7 nl;
  print 7.0 / 3.6 + 5.5 nl;
  print 1.1 / 3.6 + 7.4 nl;
  print 9.1 / 1.1 + 2.5 nl;
  print 1.9 / 6.7 + 7.2 nl;
  print 4.3 / 2.7 + 9.8 nl;
  print 8.1 / 2.6 + 8.1 nl;
  print 1.2 / 4.3 + 8.0 nl;
  print 6.5 / 3.1 + 9.5 nl;
  print 6.5 / 4.8 + 2.0 nl;
  print 5.7 / 1.7 + 4.9 nl;

  print 2.2 + 3.9 / 4.7 nl;
  print 1.0 + 5.9 / 8.1 nl;
  print 9.7 + 7.4 / 9.3 nl;
  print 4.2 + 8.4 / 3.2 nl;
  print 1.6 + 9.0 / 8.8 nl;
  print 2.1 + 1.4 / 2.8 nl;
  print 9.7 + 4.8 / 8.8 nl;
  print 4.2 + 6.6 / 4.5 nl;
  print 9.7 + 9.2 / 1.9 nl;
  print 7.5 + 9.3 / 3.1 nl;
  print 1.7 + 6.9 / 1.5 nl;
  print 8.5 + 4.8 / 7.9 nl;

  print 3.5 / 9.5 + 3.4 / 7.2 nl;
  print 1.0 / 7.2 + 1.7 / 6.4 nl;
  print 9.5 / 4.1 + 5.5 / 3.5 nl;
  print 7.9 / 6.2 + 2.3 / 1.2 nl;
  print 9.8 / 4.2 + 2.9 / 9.7 nl;
  print 3.8 / 5.9 + 3.7 / 2.6 nl;
  print 8.1 / 1.1 + 7.4 / 8.8 nl;
  print 4.0 / 5.7 + 8.1 / 6.7 nl;
  print 5.0 / 8.4 + 5.3 / 1.3 nl;
  print 2.7 / 2.8 + 2.3 / 7.1 nl;
  print 9.7 / 1.2 + 6.5 / 3.4 nl;
  print 4.0 / 8.7 + 2.7 / 6.6 nl;

  print 2.1 + 5.4 / 8.1 + 9.4 nl;
  print 1.1 + 6.4 / 1.6 + 6.1 nl;
  print 3.7 + 4.1 / 9.7 + 1.3 nl;
  print 9.5 + 8.4 / 2.3 + 2.9 nl;
  print 9.1 + 7.9 / 9.5 + 1.8 nl;
  print 1.5 + 2.3 / 4.7 + 8.5 nl;
  print 5.0 + 1.4 / 5.7 + 5.5 nl;
  print 6.8 + 5.2 / 7.4 + 3.6 nl;
  print 1.1 + 7.2 / 2.7 + 7.5 nl;
  print 7.7 + 6.6 / 8.2 + 6.8 nl;
  print 2.2 + 5.4 / 1.4 + 8.4 nl;
  print 3.6 + 2.3 / 7.1 + 7.0 nl;

}

