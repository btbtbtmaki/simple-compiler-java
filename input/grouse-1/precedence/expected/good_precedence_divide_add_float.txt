5.6
7.46812
7.44444
7.70556
10.7727
7.48358
11.3926
11.2154
8.27907
11.5968
3.35417
8.25294
3.02979
1.7284
10.4957
6.825
2.62273
2.6
10.2455
5.66667
14.5421
10.5
6.3
9.10759
0.840643
0.404514
3.8885
3.19086
2.6323
2.06714
8.20455
1.91071
4.67216
1.28823
9.9951
0.868861
12.1667
11.2
5.42268
16.0522
11.7316
10.4894
10.7456
11.1027
11.2667
15.3049
14.4571
10.9239
